Production/crab/configs/Phase2_110X/DY.txt

```diff
@@ -0,0 +1,8 @@
+Automatic
+globalTag=110X_mcRun4_realistic_v3 sampleType=MC_Phase2_110X
```

Comment:
Please, wait for #20 to be merged, because the format of crab txt files will slightly change.

Reply:
Ok, that's fine with me. I'll rebase & update Phase2 lists as soon as #20 is merged.